### ISSUE TITLE
Timeout waiting for server when launching docker containers

### ISSUE THREAD

**Issue by edwinek** [codenvy/codenvy#2558]
Hi,

I have started seeing timeouts during the inflight checks as of last night, on separate `Ubuntu 16.04` and `Debian 8` machines and was hoping for some pointers on how to get up and running.  I am attempting to start with the following command:

`docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v ~/codenvy/data:/data codenvy/cli start`

And the output is as follows:

```
WARN: Bound 'codenvy/cli' to 'codenvy/cli:5.21.0'
WARNING: No swap limit support
INFO: (codenvy cli): 5.21.0 - using docker 17.09.0-ce / native
INFO: (codenvy config): Generating codenvy configuration...
INFO: (codenvy config): Customizing docker-compose for running in a container
INFO: (codenvy start): Preflight checks
WARNING: No swap limit support
         mem (1.5 GiB):           [OK]
         disk (100 MB):           [OK]
         port 80 (http):          [AVAILABLE]
         port 443 (https):        [AVAILABLE]
         port 2181 (zookeeper):   [AVAILABLE]
         port 5000 (registry):    [AVAILABLE]
         port 23750 (socat):      [AVAILABLE]
         port 23751 (swarm):      [AVAILABLE]
         port 32000 (jmx):        [AVAILABLE]
         port 32001 (jmx):        [AVAILABLE]
         conn (browser => ws):    [OK]
         conn (server => ws):     [OK]

INFO: (codenvy start): Starting containers...
INFO: (codenvy start): Services booting...
INFO: (codenvy start): Server logs at "docker logs -f codenvy_codenvy_1"
ERROR: (codenvy start): Timeout waiting for server. Run "docker logs codenvy_codenvy_1" to inspect.
```

When opening `http://localhost:80` in a browser I get a blank screen. Viewing the source shows some HTML - the head tag, containing an unterminated link tag  `<link href="data:image/x-icon;base64,AAABAAMAMDAAAAEAIACoJQAANgAAACAgAAAB` (etc - it appears to go on forever)

The docker logs aren't showing anything obvious however there are some logback errors:

```
13:07:34,657 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.classic.net.SMTPAppende
r]
13:07:34,659 |-ERROR in ch.qos.logback.core.joran.action.AppenderAction - Could not create an Appender of type [ch.qos.logback.classic.net.SMTPAppende
r]. ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type ch.qos.logback.classic.net.SMTPAppender
        at ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type ch.qos.logback.classic.net.SMTPAppender
...
```
and 
```
13:07:34,659 |-ERROR in ch.qos.logback.core.joran.spi.Interpreter@36:72 - ActionException in Action for tag [appender] ch.qos.logback.core.joran.spi.ActionException: ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type ch.qos.logback.classic.net.SMTPAppender
        at ch.qos.logback.core.joran.spi.ActionException: ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type ch.qos.logback.classic.net.SMTPAppender
        at      at ch.qos.logback.core.joran.action.AppenderAction.begin(AppenderAction.java:76)
        at      at ch.qos.logback.core.joran.spi.Interpreter.callBeginAction(Interpreter.java:269)
 ```

I was successful in getting everything up and running on another `Ubuntu 16.04` system (that I don't have access to at the moment) on Thurs, so could something have changed in the docker image builds?

**Comment by edwinek** [comment]
I've managed to replicate this on three different Debian-based OS (Ubuntu / Debian / Mint) on wired connections using domestic router equipment (not sure if that makes any difference). I can't replicate on my work machine (Ubuntu on Wifi using enterprise router equipment).

After a some searching I found a workaround, however.

Updating resolve.conf as per this post below seems to do the trick for me:
https://github.com/moby/moby/issues/13381#issuecomment-175378071
